tests/tests.go

```diff
@@ -9,10 +9,10 @@ import (
 	. "github.com/onsi/gomega"
 
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/util/rand"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
```

Comment:
Please keep this outside of this PR. It is good to fix modules to satisfy linting, but touching modules you don't edit may cause rebase conflicts in other PRs.

Reply:
done